ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/AddMetadata.java

```diff
@@ -40,13 +40,13 @@
 
   private static final String URI = "uri";
   private static final List<String> URI_ATTRIBUTES = ImmutableList //
-      .of("app_name", "app_version", "app_update_channel", "app_build_id");
+      .of("uri", "app_name", "app_version", "app_update_channel", "app_build_id");
```

Comment:
Had some discussion with @whd about whether it's safe to include the full uri in the payload. He's fine with this for the time being to get us unstuck, but I will open an issue to discuss the reprocessing paths we need and whether there's sufficient motivation to include uri in the payload.